front/knowbaseitem.form.php

```diff
@@ -22,4 +22,3 @@
    PluginFormcreatorWizard::footer();
 
 }
-?>
```

Comment:
According to psr-2, all file must have a final line ending
http://www.php-fig.org/psr/psr-2/#2-2-files